metadatastore/doc.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import six
+import collections
+from functools import reduce
+
+_HTML_TEMPLATE = """
+<table>
+{% for key, value in document | dictsort recursive %}
+  <tr>
+    <th> {{ key }} </th>
+    <td>
+      {% if value.items %}
+        <table>
+          {{ loop(value | dictsort) }}
+        </table>
+        {% else %}
+          {% if key == 'time' %}
+            {{ value | human_time }}
+          {% else %}
+            {{ value }}
+          {% endif %}
+        {% endif %}
+    </td>
+  </tr>
+{% endfor %}
+</table>
+"""
```

Comment:
Is this the same repr that was in the old document.Document? If not, @danielballan are you still :+1: on this html table repr?  I remember at some point recently you were not happy with it.